libs/model/src/featurelayer.cpp

```diff
@@ -83,11 +83,138 @@ TileFeatureLayer::TileFeatureLayer(
 
 TileFeatureLayer::~TileFeatureLayer() = default;
 
+namespace
+{
+
+/* Check that starting from a given index, the parts of an id composition
```

Comment:
Docstrings should start with `/**` for consistency (I think this way they get picked up as such by the IDE or Doxygen)

Reply:
Noted!